### PR TITLE
Updating tests to work when one directory deeper

### PR DESCRIPTION
**Pull request by fidian**
Moves all tests into "spec/lib" so we can make "spec/route".

### CODE REVIEW

**Comment by fidian** [comment]
Merged master and removed the conflict.

**Comment by fidian** [comment]
I've added a test for the route as well.  This brings up code coverage!

File: route/index.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
-module.exports = {
-    get: function (req, res, next) {
-        res.setHeader("Content-Type", "text/plain");
-        res.send("API running " + (new Date()) + "\n");
-        next();
-    }
+module.exports = () => {
```
Comment (AbsentSemicolon) [inline route/index.js]: Should there be a "use strict"; at the top of the file?

File: spec/mock/request.js [hunk 2]
```diff
@@ -0,0 +1 @@
+module.exports = {};
```
Comment (AbsentSemicolon) [inline spec/mock/request.js]: Should there be a "use strict"; at the top of the file?

File: spec/mock/response.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+var response;
```
Comment (AbsentSemicolon) [inline spec/mock/response.js]: Asking again, just to make sure, should there be a "use strict"?

File: spec/route/index.spec.js [hunk 4]
```diff
@@ -0,0 +1,33 @@
+describe("route: /", () => {
```
Comment (AbsentSemicolon) [inline spec/route/index.spec.js]: Last place for "use strict".

**Comment by fidian** [comment]
Added "use strict" - thanks for catching that!

**Comment by AbsentSemicolon** [comment]
No problem. Reviewed and looks good.

**Pull request merged**